dev-libs/tree-sitter-python/Manifest

```diff
@@ -0,0 +1 @@
+DIST tree-sitter-python-0.19.9999_p20211112.tar.gz 141155 BLAKE2B ee524faca6bebf6c7f27d34ba418c4494877d6cf33a17b4ac0e5c148db58a8e17b2ec8ba992139b72e1fc88c9c93bcdb6a8cbd4a47b9cc65ad3097e08b4ee339 SHA512 25d5b3f8bac1daf8c9971278b8c209f2d953310cd31afd2e375d48354845930e553d0067dd9c0e93565d1996a803d0b5445f48720b0bc7f3d38e73f40cdbbc90
```

Comment:
What's going on with the version number year?

We normally do e.g. 0.19_p2021112 for a snapshot of the last commit with date 2021112 after release 0.19.

Reply:
ok.

Although, "after 0.19 release" (well, actually it was named "rust-0.19.1") is true for python grammar.
(btw, would it be more correct to make `0.19.1_p*` then?)

But I also picked same pseudo-version for "haskel" grammar, whilere there were no 0.19 release at all. There was pretty old `v0.0` release, but taken commit contains 0.19+ compatible grammar and stuff.
And I picked `0.19` there to only make eclass to not explode (without #23216 merged).

I plan to change it to `0_p*` then, and utilize `TS_ABI` from #23216, would it be ok?